tools/ci_connector_ops/setup.py

```diff
@@ -10,21 +10,31 @@
     "PyYAML~=6.0",
     "GitPython~=3.1.29",
     "pandas~=1.5.3",
+    "pandas-gbq~=0.19.0",
     "pydantic~=1.10.4",
     "fsspec~=2023.1.0",
     "gcsfs~=2023.1.0"
 ]
 
+TEST_REQUIREMENTS = [
+    "pytest~=6.2.5",
+    "pytest-mock~=3.10.0",
+]
+
 
 setup(
-    version="0.1.3",
+    version="0.1.5",
```

Comment:
Anticipating a version bump. 0.1.4 is in https://github.com/airbytehq/airbyte/issues/2177